dashboard/app/views/reports/search_for_teachers.html.haml

```diff
@@ -1,3 +1,8 @@
+:css
```

Comment:
Given the !important, I don't want to move it to a common location as is. Added a TODO to clean up the CSS and move it to a shareable location.